sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala

```diff
@@ -154,6 +155,14 @@ object ScalaNativePluginInternal {
       }
       libs
     },
+    nativeMode := "debug",
+    nativeLinkerReporter := tools.LinkerReporter.empty,
+    nativeOptimizerReporter := tools.OptimizerReporter.empty,
+    nativeLogger := streams.value.log,
+    nativeGC := "boehm"
+  )
```

Comment:
They are, I think you've been tricked by the folded diff. All the keys in global scope are:
`nativeWarnOldJVM`, `nativeClang`, `nativeClangPP`, `nativeCompileOptions`, `nativeLinkingOptions`, `nativeMode`, `nativeLinkerReporter`, `nativeOptimizerReporter`, `nativeLogger`, and `nativeGC`.